### PR TITLE
Update hybrid-azure-ad-join-intune-connector.md

### PR DESCRIPTION
**Pull request by msbemba**
Added intune administrator role to the note

Fixes https://github.com/MicrosoftDocs/memdocs/issues/4354

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 please copy review

File: autopilot/tutorial/user-driven/hybrid-azure-ad-join-intune-connector.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ For an overview of the Windows Autopilot user-driven Microsoft Entra hybrid join
 
 > [!NOTE]
 >
-> - The Global administrator role is a temporary requirement at the time of installation.
+> - The Global administrator role or an Intune administrator role is a temporary requirement at the time of installation.
```
Comment (JohanFreelancer9) [inline autopilot/tutorial/user-driven/hybrid-azure-ad-join-intune-connector.md:103]:
```suggestion
> - The Global Administrator role or an Intune Administrator role is a temporary requirement at the time of installation.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@frankroj Please review and sign off this PR. Thanks

**Comment by frankroj** [comment]
I will need to verify that the addition is correct. Just because Intune admin appears to work doesn't mean it will always work or is supported.

**Comment by frankroj** [comment]
Made some adjustments to the wording. Removed the specific account names and just said whatever account used is only temporary. Also made some other small improvements in the surrounding text.

**Comment by frankroj** [comment]
#sign-off

**Pull request merged**